### DISCUSSION TITLE
Ability to select different database (mariaDB)

### DISCUSSION THREAD

**Discussion by lorenzofattori** [teslamate-org/teslamate#1688] [Ideas]
I'm running Teslamate on a raspberry PI together with home assistant, mosquitto and other tools in containers.
My Home assistant is set up to store data on a MariaDB database stored on a synology diskstation, in order to reduce read and write on the SD of the Raspberry.

It would be nice to have the ability to use MariaDB for TeslaMate as well, so less data will be written on the SD.

Thoughts?

**Comment by adriankumpf** [comment]
See https://github.com/adriankumpf/teslamate/issues/710

**Reply by lorenzofattori** [reply]
thank you
